k8s/predict/stage-airqo-predict-job.yaml

```diff
@@ -17,7 +17,7 @@ spec:
           schedulerName: default-scheduler
           containers:
           - name: stage-airqo-predict-job
-            image: us.gcr.io/airqo-250220/airqo-stage-predict-job:latest
+            image: us.gcr.io/airqo-250220/stage-airqo-predict-job:latest
```

Comment:
Hi @123MwanjeMike, it looks like this change doesn't match the current image name. Please advice.

<img width="611" alt="Screen Shot 2022-05-19 at 7 37 25 AM" src="https://user-images.githubusercontent.com/15224992/169207059-452bd250-5daf-4fec-95c2-7b259c153f9a.png">

Reply:
Thanks for catching this @danielogen.
Both images exist, although `stage-airqo-predict-job` is the one currently pushed to the registry by the staging pipeline so its the one with the recent changes.
We could perhaps remove the duplicate old one.
![image](https://user-images.githubusercontent.com/41787587/169223483-6ee369f8-cf81-46ae-858b-0de6905225ab.png)

Reply:
Prefect @123MwanjeMike , I have gone ahead and removed the duplicate